en/setup/config_initial.md

```diff
@@ -1,22 +1,34 @@
-# Initial Configuration
+# Initial Configuration & Setup
 
-Before starting to develop on PX4, the system should be configured initially with a default configuration to ensure the hardware is set up properly and is tested. The video below explains the setup process with [Pixhawk hardware](https://docs.px4.io/en/flight_controller/pixhawk_series.html) and [QGroundControl](../qgc/README.md). A list of supported reference airframes is [here](../airframes/architecture.md).
+Before starting to work on PX4 we recommend that developers obtain the basic equipment described below (or similar), and initially use a "default" configuration for their [airframe](../airframes/airframe_reference.md).
 
-> **Info** [Download the DAILY BUILD of QGroundControl](https://docs.qgroundcontrol.com/en/releases/daily_builds.html) and follow the video instructions below to set up your vehicle. See the [QGroundControl Tutorial](../qgc/README.md) for details on mission planning, flying and parameter setting.
+> **Tip** PX4 can be used with a much wider range of equipment, but new developers will benefit from going with one of the standard setups, and a Taranis RC plus a Note 4 tablet make up for a very inexpensive field kit.
 
-A list of setup options is below the video.
+## Basic Equipment
 
-{% youtube %}https://www.youtube.com/watch?v=91VGmdSlbo4&rel=0&vq=hd720{% endyoutube %}
+The equipment below is highly recommended:
+* A Taranis Plus remote control for the safety pilot (or equivalent)
+* A development computer
+  * MacBook Pro or Air with OSX 10.10 or newer
+  * Modern laptop with Ubuntu Linux (14.04 or newer)
+* A ground control station device
+  * Any MacBook or Ubuntu Linux laptop (can be the development computer)
+  * Samsung Note 4 or equivalent / more recent Android tablet
+  * iPad (requires Wifi telemetry adapter)
+* Safety glasses
+* For multicopters - tether for more risky tests
+
+## Vehicle Configuration
+
+Download the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/en/releases/daily_builds.html) for your development platform.
 
-## Radio Control Options
```

Comment:
@bkueng I deleted all this stuff. The relevant configuration is all in the PX4 user guide as linked